i created a navagation bar

MrFrostUg opened the pull request:
i created a navigation bar and used album templates for my index page layout

MrFrostUg committed:
i created a navagation bar

MrFrostUg committed:
removed a default meta author from bootstrap and replaced with my name. removed style from html file

MrFrostUg committed:
created a function to add a logo to navigation bar

MrFrostUg committed:
i added hoover effect on the icon

MrFrostUg committed:
bootstrap icons installed

hi-coders commented on style.css:
```diff
@@ -0,0 +1,100 @@
+  text-decoration: none;
+}
+
+.Parent {
```
lower case!

MrFrostUg replied:
i have not understood by what you mean here please, what should i make lower case please?

hi-coders commented on style.css:
```diff
@@ -0,0 +1,100 @@
+  grid-gap: 15px;
+}
+
+.child1 {
```
child1 child2 are not good names

MrFrostUg replied:
resolved to parent, left and right grid respectively

hi-coders commented on style.css:
```diff
@@ -0,0 +1,100 @@
+  height: 50vh;
+};
+.navbar-logo {
+width: 10px;
```
use indentation properly

MrFrostUg replied:
resolved by formatting using the respective default language formatt

MrFrostUg committed:
*removed json files
*formatted code for proper indentetation
*changed from parent,child1 and child2  to parent-grid, left-grid and right-grid respectively
*resolved some camel case issue from Parent to parent

MrFrostUg committed:
deleted functions

MrFrostUg committed:
restructuring the code

MrFrostUg committed:
html, css, javascrip all code  restructing

MrFrostUg committed:
created a navigation bar

MrFrostUg committed:
added Css to navbar

MrFrostUg committed:
i have created a new navigation bar

MrFrostUg committed:
pasted my code to chat gpt to organize it

MrFrostUg committed:
made changes to logo and footer

MrFrostUg committed:
added student cards

MrFrostUg committed:
added a student add button

MrFrostUg committed:
created teacher,students object classses

MrFrostUg committed:
formatted code for good indenting

MrFrostUg committed:
removed redundant code

MrFrostUg committed:
alll code deleted

MrFrostUg committed:
navigation bar added and cards to home page

MrFrostUg committed:
navigation bar and cards added to the home page

MrFrostUg commented:
navigation bar added and cards to the home page

MrFrostUg committed:
cdn links added

MrFrostUg committed:
created files for the codes

MrFrostUg committed:
added student, classes,teachers data objects

MrFrostUg committed:
added elements to html using modules but for some reason i dont know nothing is displaying when i run live server

MrFrostUg committed:
deleted

MrFrostUg committed:
created index html

MrFrostUg committed:
added welcome message to home page
